src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -26,6 +26,8 @@ public class PlayerTabList {
     private final Velocitab plugin;
     private final ConcurrentLinkedQueue<TabPlayer> players;
     private final ConcurrentLinkedQueue<String> fallbackServers;
+    private int indexheader = 0;
```

Comment:
```suggestion
    private int headerIndex = 0;
```
style/camelCase variable names